src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,103 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class StringCalculator {
+
+    public static final String SPACEBAR = " ";
+    public static final String IS_NUMBER_REGEX = "[0-9]+";
+    public static final String IS_OPERATOR_REGEX = "[-+*/]";
+    public static final String IS_NULL_ERROR_MESSAGE = "입력 값이 null이거나 빈 공백 문자입니다.";
+    public static final String IS_NOT_OPERATOR_ERROR_MESSAGE = "사칙연산 기호가 아닌 문자가 포함되어 있습니다.";
+
+    public int execute(String input) {
+        validateInput(input);
+        String[] inputs = input.split(SPACEBAR);
+        List<Integer> numbers = parsingNumber(inputs);
+        List<String> operations = parsingOperation(inputs);
+        return input(numbers, operations);
+    }
+
+    private void validateInput(String input) throws IllegalArgumentException {
+        if(isEmpty(input)) {
+            throw new IllegalArgumentException(IS_NULL_ERROR_MESSAGE);
+        }
+        if(isNotOperator(input)) {
+            throw new IllegalArgumentException(IS_NOT_OPERATOR_ERROR_MESSAGE);
+        }
+    }
+
+    private boolean isEmpty(String input) {
+        if(Objects.isNull(input) || input.trim().isEmpty()) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return Objects.isNull(input) || input.trim().isEmpty(;
```

와 같이도 표현할 수 있을 것 같아요.